src/modules/75-cd/components/PipelineSteps/ShellScriptStep/shellScriptTypes.ts

```diff
@@ -38,18 +44,23 @@ export const scriptOutputType: SelectOption[] = [
   { label: 'Secret', value: 'Secret' }
 ]
 
-export interface ShellScriptStepVariable {
+export interface ScriptStepVariable {
```

Comment:
NIT: Make the interface generic

```suggestion
export interface ScriptStepVariable<T = 'String' | 'Number'> {
  value: number | string
  id: string
  name?: string
  type?: T
}

export type ScriptOutputStepVariable = ScriptStepVariable<'String'>

export type ShellScriptStepVariable = ScriptStepVariable<'String' | 'Number' | 'Secret'>

export type ShellScriptOutputStepVariable = ScriptStepVariable<'String' | 'Secret'>
```

Reply:
sure, Thank you !